acceptance/test_run_scripts/555-puppetserver-service/reboot-foss-rhel7.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+SCRIPT_PATH=$(pwd)
+BASENAME_CMD="basename ${SCRIPT_PATH}"
+SCRIPT_BASE_PATH=`eval ${BASENAME_CMD}`
+if [ $SCRIPT_BASE_PATH = "555-puppetserver-service" ]; then
+  cd ../../..
+fi
+
+export PACKAGE_BUILD_VERSION=1.0.4.SNAPSHOT.2015.03.24T0146
```

Comment:
Won't this pin the behavior of this script to a specific build version?  I ask because this seems like something intended to be generally applicable to different build versions.